bears/dart/DartLintBear.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class DartLintBear(LocalBear, Lint):
+    executable = 'dartanalyzer'
+    output_regex = re.compile(
+        r'(?P<severity>\[error\]|\[warning\])\s(?P<message>.*)\('
+        r'(?P<file_name>.+),\s[l][i][n][e]\s(?P<line>.*),'
+        r'\s[c][o][l]\s(?P<column>.*)\)')
+    severity_map = {
+        "[error]": RESULT_SEVERITY.MAJOR,
+        "[warning]": RESULT_SEVERITY.NORMAL}
+
+    def run(self, filename, file):
+        '''
+        Checks the code with `dart-linter`. Note: This bear expects dartanalyzer
+        to be on your PATH. Please ensure /path/to/dart-sdk/bin is in your PATH.
```

Comment:
just enclose it in those:

    ``PATH``

(I don't think the dollar is needed, that's bashy)